bot/cogs/sticky.py

```diff
@@ -197,44 +187,85 @@ async def remove_sticky_message(
                     color=discord.Color.red(),
                     title="❌ Sticky message not exist",
                     description=f"Tidak ada sticky message pada channel {channel.mention}",
-                    timestamp=datetime.now(),
                 )
             else:
                 data = dict(res[0])
-
-                sticky = await channel.fetch_message(data["message_id"])
-                await sticky.delete()
+                try:
+                    sticky = await channel.fetch_message(data["message_id"])
+                    await sticky.delete()
+                except discord.errors.NotFound:
+                    pass
 
                 async with self.db_pool.acquire() as conn:
                     await conn.execute("DELETE FROM sticky WHERE channel_id = $1;", channel.id)
 
                 self.sticky_data.pop(channel.id)
 
-                embed = discord.Embed(
+                await send_interaction(
+                    interaction,
                     color=discord.Color.green(),
                     title="✅ Sticky message removed successfully",
                     description=f"Berhasil menghapus sticky message pada channel {channel.mention}",
-                    timestamp=datetime.now(),
                 )
-
         else:
-            embed = discord.Embed(
+            await send_interaction(
+                interaction,
                 color=discord.Color.red(),
                 title="❌ You Don't Have Permission To Delete Sticky Message",
                 description=f"Permission Manage Channel Dibutuhkan",
-                timestamp=datetime.now(),
             )
 
-        embed.set_footer(
-            text=f"{str(interaction.user)}",
-            icon_url=interaction.user.display_avatar.url,
-        )
+    @app_commands.command(name="re-send", description="Re-send sticky message to channels.")
+    @app_commands.describe(channel="Target Channel")
+    async def fix_sticky_message(
+        self,
+        interaction: Interaction,
+        channel: Union[discord.TextChannel, discord.ForumChannel, discord.Thread],
+    ) -> None:
+        await interaction.response.defer()
+        if interaction.permissions.manage_channels:
+            async with self.db_pool.acquire() as conn:
+                res = await conn.fetch(
+                    "SELECT * FROM sticky WHERE channel_id = $1;",
+                    channel.id,
+                )
 
-        await interaction.followup.send(embed=embed)
+            if not res:
+                await send_interaction(
+                    interaction,
+                    color=discord.Color.red(),
+                    title="❌ Sticky message not exist",
+                    description=f"Tidak ada sticky message pada channel {channel.mention}",
+                )
+            else:
+                data = dict(res[0])
+                try:
+                    await channel.fetch_message(data["message_id"])
+                    pass
```

Comment:
```suggestion
                try:
                    await channel.fetch_message(data["message_id"])
                    return
```

Use `return` instead to skip the entire logic in the function